packages/backend/src/api/resources/JSON.ts

```diff
@@ -309,3 +309,37 @@ export interface DeletedObjectJSON {
   slug?: string;
   deleted: boolean;
 }
+
```

Comment:
Can we place these in a webhooks.ts file? 
Also, I realiase that the JSONs exist in the backend pkg, but do we place this in backend or in types?

cc @dimkl

Reply:
Should we use `webhooks.ts` or `webhooks.d.ts` ? Let's extract the types. 
I think it's `backend` specific and depends on the `JSON` types that are only accessible from this package. To move the `Webhook` type to `types` we should also move the `JSON` types, which i think we shouldn't.

Reply:
> Should we use webhooks.ts or webhooks.d.ts ?
Should be webhooks.ts, TS will generate the d.ts file during the build step. 

> To move the Webhook type to types we should also move the JSON types, which i think we shouldn't.
Let's stick with @clerk/backend though

@desiprisg Let's move these to a webhooks.ts file  and then we're good to go. Thanks both :)

Reply:
@desiprisg Once this is merged and release, we should also add a "typescript support" section to the webhooks docs here: https://clerk.com/docs/integration/webhooks